src/packets.c

```diff
@@ -105,15 +105,15 @@ struct rte_mbuf **pg_packets_append_str(struct rte_mbuf **pkts,
 					const char *str)
 {
 	PG_PACKETS_OPS_STR(pkts, pkts_mask, str, append)
-	return pkts;
+		return pkts;
```

Comment:
`return` and `PG_PACKETS_OPS_STR()` should be on the same level of indentation.
We can add a ";" at the end of `PG_PACKETS_OPS_STR()` so it do not fools the indent.